examples/annotate-config/README.md

```diff
@@ -0,0 +1,22 @@
+# Annotate Config
+
+The `annotate-config` KRM config function adds an annotation to configuration.
+This example invokes the annotate-config function using declarative
```

Comment:
This sentence is repeated in all examples and doesn't convey much to the user. Please remove everywhere.

Reply:
Done in all outstanding blueprints PRs